apps/settings/lib/Controller/UsersController.php

```diff
@@ -250,13 +263,16 @@ public function usersList(INavigationManager $navigationManager, ISubAdmin $subA
 	 */
 	#[AuthorizedAdminSetting(settings:Users::class)]
 	public function setPreference(string $key, string $value): JSONResponse {
-		$allowed = ['newUser.sendEmail', 'group.sortBy'];
-		if (!in_array($key, $allowed, true)) {
+		$allowedAppValues = ['newUser.sendEmail', 'group.sortBy'];
```

Comment:
```suggestion
		$allowedKeys = ['newUser.sendEmail', 'group.sortBy'];
```
This would be more precise, no? Alternatively just inline the array, since the variable is only used once.